test/checks/fuerte/distro-to-install.py

```diff
@@ -1,16 +1,17 @@
 #!/usr/bin/env python
 
+from __future__ import print_function
```

Comment:
This is not necessary in this case (see http://wiki.ros.org/python_2_and_3_compatible_code#Print_is_now_a_function).

Same below for many other cases.